clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/kafka/KafkaInputRecordReader.java

```diff
@@ -294,15 +305,32 @@ public void close() {
   @Override
   public float getProgress() {
     // not most accurate but gives reasonable estimate
-    return ((float) (currentOffset - startingOffset + 1)) / maxNumberOfRecords;
+    if (totalRecordsToRead <= 0) {
+      return 1.0f;
```

Comment:
maybe `Float.NaN` is more appropriate

Reply:
Not sure that will break MR. Keeping update change from 0 to 1 as suggested in prev comment